src/snarky.js

```diff
@@ -1,3 +1,4 @@
+import './bindings/crypto/bindings.js';
```

Comment:
so, here's how this is supposed to work:

the new `bindings.ts` file imported here is written in TS, and contains code integrated with our normal codebase. It is exposed to JSOO by populating a global variable with an object. It gets imported as the first thing here so that the global variable is ready by the time JSOO code gets executed.

It may not be the cleanest thing to have an implicit dependency like that, but it's a huge improvement over the status quo, where all JS code used in JSOO has to be part of an ugly file containing outdated syntax which can use neither TS nor normal imports.